packages/plugins/Zoom/src/components/ZoomSlider.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div>
+    <v-slider
+      v-model="zoomLevelRange"
+      vertical
+      class="polar-zoom-slider"
+      :min="minZoom"
+      :max="maxZoom"
+      :title="$t('common:plugins.zoom.slider')"
+    ></v-slider>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { mapGetters, mapActions } from 'vuex'
+
+export default Vue.extend({
+  name: 'ZoomSlider',
+  props: {
+    minZoom: {
+      type: Number,
+      default: 0,
+    },
+    maxZoom: {
+      type: Number,
+      default: 9,
+    },
+  },
+  computed: {
+    ...mapGetters('plugin/zoom', ['zoomLevel']),
+    zoomLevelRange: {
+      get(): number {
+        return this.zoomLevel
+      },
+      set(value: number): void {
+        this.setZoomLevel(value)
+      },
+    },
+  },
+  methods: {
+    ...mapActions('plugin/zoom', ['setZoomLevel']),
+  },
+})
+</script>
+
+<style lang="scss">
```

Comment:
Can some of the styles be scoped?

Reply:
unfortunately no, but I added a specific class for the zoom slider so that this style will not be applied to other v-sliders in the app.